src/api.c

```diff
@@ -537,3 +551,64 @@ API int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd)
 
 	return 0;
 }
+
+/* NOTE - function header comment in include/seccomp.h */
+int seccomp_alloc_notifications(struct seccomp_notif **req,
+				struct seccomp_notif_resp **resp)
+{
+	static struct seccomp_notif_sizes sizes = {};
```

Comment:
This information should probably live in `src/system.c`.  See the other comments in this function.